### DISCUSSION TITLE
Problem rendering the website in #French

### DISCUSSION THREAD

**Discussion by Kkhurana007** [quarto-dev/quarto-cli#10445] [Q&A] [answered]
### Description

Quarto version:- 1.5.40
OS- Win 10

 I can see the English and French content of the website when I use Quarto's Metadata function: https://quarto.org/docs/projects/quarto-projects.html#metadata-merging

Here is what I see from my laptop when I try to render things: 

![image](https://github.com/user-attachments/assets/d41c693d-3b6f-4acc-b55a-68e13822ca24)

So, I want to see things in English first, and then when I click the button for French (in the top right or bottom right), I want to see things in the other language (French, for example).

Here is the link to the git repository: https://github.com/Kkhurana007/sustainable-future

Here is the link to the repository that I found useful to build this project: https://github.com/jtrecenti/beaeju.com.br/

Any help will be greatly appreciated 🙏

@mcanouil @beatrizmilz

**Comment by mcanouil** [comment, marked as answer]
I suggest to read the other threads on the matter.

For example:
- https://github.com/quarto-dev/quarto-cli/issues/275
- https://github.com/quarto-dev/quarto-cli/discussions/10246
- https://github.com/quarto-dev/quarto-cli/discussions/5262
